lib/addons/topics-api.ts

```diff
@@ -0,0 +1,35 @@
+import OptableSDK from "../sdk";
+
+declare module "../sdk" {
+    export interface OptableSDK {
+        tryTopicsAPI: () => void;
+        getTopics: () => Promise<void>;
+    }
+}
+
+declare global {
+    interface Document {
+        browsingTopics: () => Promise<any[]>;
```

Comment:
I'd recommend to:
1/ mark it optional
2/ type the returned array rather than `any[]`

Reply:
1. Won't build
lib/addons/topics-api.ts(27,35): error TS2722: Cannot invoke an object which is possibly 'undefined'.
lib/addons/topics-api.ts(27,35): error TS18048: 'document.browsingTopics' is possibly 'undefined'

2. fixed

Reply:
1/ you need to typeof !== undefined check it as well, `x in y` only checks for property being set, but it could be set with undefined